google-auth-provider/tool.gpt

```diff
@@ -12,6 +12,7 @@ Credential: ../placeholder-credential as google-auth-provider
     "icon": "/admin/assets/google_icon_small.png",
     "iconDark": "/admin/assets/google_icon_small.png",
     "link": "https://google.com/",
+    "postgresTablePrefix": "google_",
```

Comment:
We should probably add the underscore in https://github.com/obot-platform/obot/pull/2346/files#diff-1d620309aefdeee90bc0b70186dec69b901dff7d8dae034d0124565d98dc17f6R251 so that the metadata field is the same as the value the auth provider uses for the oauth2-proxy config.

Reply:
fixed here, enterprise tools, and in oauth2-proxy